prime-router/src/test/kotlin/azure/ApiSearchTest.kt

```diff
@@ -0,0 +1,225 @@
+package gov.cdc.prime.router.azure
+
+import assertk.assertThat
+import assertk.assertions.contains
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNotNull
+import assertk.assertions.isNull
+import gov.cdc.prime.router.db.ReportFileApiSearch
+import org.jooq.Condition
+import org.jooq.Field
+import org.jooq.Name
+import org.jooq.Record2
+import org.jooq.Row2
+import org.jooq.SortOrder
+import org.jooq.Table
+import org.jooq.TableField
+import org.jooq.TableOptions
+import org.jooq.impl.DSL
+import org.jooq.impl.DSL.name
+import org.jooq.impl.SQLDataType
+import org.jooq.impl.TableImpl
+import org.jooq.impl.UpdatableRecordImpl
+import java.time.OffsetDateTime
+import kotlin.test.Test
+
+@Suppress("UNCHECKED_CAST")
+class ApiSearchTest {
+
+    class TestRecord : UpdatableRecordImpl<TestRecord>(TestPojo.TEST_POJO), Record2<String, OffsetDateTime> {
```

Comment:
This is strictly a test instance of the JOOQ classes to do some testing without a db